docs/introduction/tasks.md

```diff
@@ -70,168 +70,13 @@ i4ir3kvv62odsfrhko4y8w2an
-The name of the parent directory is a package-level variable that applies to all
-downloads with a session. It prevents the user from cluttering the current
-working directory with a large number of downloaded files. However, the user can
-change this behavior by setting the parent directory name using the
-`inductiva.set_output_dir` function.
-
-```python
->>> import inductiva
-# set the parent directory name
->>> inductiva.get_output_dir()
-'inductiva_output'
->>> inductiva.set_output_dir("my_vault")
->>> inductiva.get_output_dir()
-'my_vault'
-
-# download the outputs again
->>> output_dir = task.download_outputs()
-Downloading simulation outputs to my_vault/i4ir3kvv62odsfrhko4y8w2an/output.zip.
-100%|██████████████████████████████████████| 5.04M/5.04M [00:00<00:00, 13.3MB/s]
-Uncompressing the outputs to my_vault/i4ir3kvv62odsfrhko4y8w2an/
->>> print(ouptut_dir)
-PosixPath('my_vault/i4ir3kvv62odsfrhko4y8w2an')
-
-# Unset the parent directory name so that all downloads are done to the current
-# working directory
->>> inductiva.set_output_dir("None")
->>> output_dir = task.download_outputs()
-Downloading simulation outputs to i4ir3kvv62odsfrhko4y8w2an/output.zip.
-100%|██████████████████████████████████████| 5.04M/5.04M [00:00<00:00, 13.3MB/s]
-Uncompressing the outputs to i4ir3kvv62odsfrhko4y8w2an/
->>> print(ouptut_dir)
-PosixPath('i4ir3kvv62odsfrhko4y8w2an')
-```
-
-The `download_outputs` method of the `Task` class has the option to specify the
-name of the folder where the outputs are downloaded inside the parent directory.
-
-```python
-# reset the parent directory name to the default
->>> inductiva.set_output_dir("inductiva_outputs")
->>> output_dir = task.download_outputs(output_dir="my_outputs")
-Downloading simulation outputs to inductiva_output/my_outputs/output.zip.
-100%|██████████████████████████████████████| 5.04M/5.04M [00:00<00:00, 13.3MB/s]
-Uncompressing the outputs to inductiva_output/my_outputs/
->>> print(ouptut_dir)
-PosixPath('inductiva_output/my_outputs')
-```
-
-In addition to controlling how downloaded files are organized, the user can also
-configure which files to download. By default, all files produced by the task are
-downloaded. However, the user can download only a subset of those files by
-specifying the filenames of interest:
-
-```python
-# Download only the files of interest
->>> output_dir = task.download_outputs(filenames=["stdout.txt", "stderr.txt"]
-                                       output_dir="my_outputs")
-Downloading simulation outputs to inductiva_output/my_outputs/output.zip.
-100%|██████████████████████████████████████| 1.04k/1.04k [00:00<00:00, 671kB/s]
-Uncompressing the outputs to inductiva_output/my_outputs/
-```
-In this case, we can confirm that only the files `stdout.txt` and `stderr.txt`
-were indeed downloaded:
-
-```bash
-$ ls -1 inductiva_output/my_outputs
-stderr.txt
-stdout.txt
-```
-
 ### Synchronous tasks
```

Comment:
I think we should highlight the nature of the task being async by default, since that is what is cool about it and permits much of the functionality. Then we can comment on making it sync.